JortPob/Model/COLLISIONtoOBJ.cs

```diff
@@ -93,5 +93,79 @@ public static Obj COLLISIONtoOBJ(List<Tuple<Node, Mesh>> collisions, Obj.Collisi
 
             return obj;
         }
+
+        public static Obj COLLISIONtoOBJ(TES3.VecMesh collisions, Obj.CollisionMaterial material)
+        {
+            Obj obj = new();
+
+            for (int mIdx = 0; mIdx < collisions.Count; mIdx++)
+            {
+                var mesh = collisions[mIdx];
```

Comment:
nit: this could be a `foreach` over `collisions`: `foreach (var mesh in collisions) {`